back/tsconfig.json

```diff
@@ -59,14 +59,14 @@
     // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
     // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */
     /* Source Map Options */
-    // "sourceRoot": "",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
+    "sourceRoot": "/",                      /* Specify the location where debugger should locate TypeScript files instead of source locations. */
```

Comment:
Are you sure this is necessary? It seems weird but I don't know why you need this.

Reply:
We could try without that, but it's a prerequisite in the sentry documentation